recipe/meta.yaml

```diff
@@ -10,25 +10,19 @@ source:
   sha256: 52dc309da50ea06d02d4d8364a8e1f9fb8da2f1c30fdd65eff119b81ee434bd4
   patches:
     # - 487.patch
-    - 0001-make-protobuf-linkable.patch
+    # - 0001-make-protobuf-linkable.patch
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win or aarch64]
   script:
   # install buf
     - set -e
     - set -x
-    - env | sort
-    - export GOBIN="${BUILD_PREFIX}/bin"
-    - export GO111MODULE=on
-    - go install github.com/bufbuild/buf/cmd/buf@v1.5.0
-    - which buf
-  # regenerate substrait to match the linked protobuf version
-    - python scripts/update_substrait.py
-    - rm -rf third_party/google/protobuf
-    - export USE_SYSTEM_PROTOBUF=true
-    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  # nuke local `.git` so we can override the SHA bundled as the version
+    - rm -rf .git
+  # this needs to match the upstream release commit SHA for extension lookup to work
+    - export SETUPTOOLS_SCM_PRETEND_VERSION="da9ee490d"
```

Comment:
This looks a bit weird, why isn't this the package version anymore?

Reply:
Because extensions in duckdb are hosted with a URL scheme like

`http://extensions.duckdb.org/{SHA1}/linux_amd64/{extension_name}.duckdb_extension.gz`

and so if the setuptools version doesn't match the upstream SHA they aren't accessible.

Reply:
Pretty sure the hash is stored in `SETUPTOOLS_SCM_PRETEND_HASH`, `SETUPTOOLS_SCM_PRETEND_VERSION` should still be `{{ version }}`, see https://github.com/duckdb/duckdb/blob/5c9a3d68662075ce3d5be9edfe02b524036f4fdb/scripts/package_build.py#L108-L134